Modules/Services/Sources/Models/Block/BlockConverters/AlignmentConverters.swift

```diff
@@ -5,7 +5,7 @@ import ProtobufMessages
 public extension Anytype_Model_Block.Align {
     var asBlockModel: LayoutAlignment? {
         switch self {
-        case .left: return .left
+        case .left, .justify: return .left
```

Comment:
fallback for left for now 
https://linear.app/anytype/issue/IOS-2804/support-justify-text-align